src/qt/res/css/crownium.css

```diff
@@ -1290,17 +1290,6 @@ padding-top:20px;
 padding-bottom:18px;
 }
 
-QDialog#SendCoinsDialog .QCheckBox#checkUseInstantSend { /* InstantSend Checkbox */
-color:#555555;
-font-weight:bold;
-background: qradialgradient(cx:0.5, cy:0.5, radius: 0.5, fx:0.5, fy:0.5, stop:0 rgba(248, 246, 246, 128), stop: 1 rgba(0, 0, 0, 0));
-border-radius:5px;
-padding-top:20px;
-padding-bottom:18px;
-margin-left:10px;
-margin-right:20px;
```

Comment:
`margin-right` should now be applied to `QDialog#SendCoinsDialog .QCheckBox#checkUsePrivateSend` to keep PS checkbox away from "Balance:" label, see https://github.com/UdjinM6/dash/commit/b9d834da52f87464538edaef8c28395dc414ed88.

Reply:
Thanks, added it :)